README.md

```diff
@@ -15,33 +15,31 @@ A basic example of how to configure glog to send logged errors to Airbrake.io:
 package main
 
 import (
-	"errors"
+    "errors"
```

Comment:
Please revert tab-to-space changes.

Reply:
`md` Linter is throwing a warning `no-hard-tabs`. To fix those warnings we shifted to spaces from tabs

Reply:
Golang doesn't use spaces to indent its code. Everyone who copies this example, will copy it with spaces.